telemetry.go

```diff
@@ -0,0 +1,32 @@
+package gocron
+
+import (
+	"sync"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	jobLatency    *prometheus.HistogramVec
+	telemetryOnce sync.Once
+
+	// Latency buckets for job latency metrics. This can be overridden on the
+	// application side
+	LatencyBuckets = prometheus.ExponentialBuckets(1, 2.5, 15)
+)
+
+func initTelemetry() {
+	telemetryOnce.Do(func() {
+		jobLatency = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+			Name:    "gocron_job_latency_milliseconds",
+			Help:    "time taken to execute a job in milliseconds",
+			Buckets: LatencyBuckets,
+		}, []string{"job_name", "scheduler_name"})
```

Comment:
I think the documentation should note how your metrics will show up if you use anonymous functions:

```go
package main

import (
	"fmt"
	"reflect"
	"runtime"
)

func myFunc() {}

func main() {
	// this is how getFunctionName() in our lib determines the name
	fmt.Println(runtime.FuncForPC(reflect.ValueOf(func() {}).Pointer()).Name())
	fmt.Println(runtime.FuncForPC(reflect.ValueOf(func() {}).Pointer()).Name())
	fmt.Println(runtime.FuncForPC(reflect.ValueOf(myFunc).Pointer()).Name())
	fmt.Println(runtime.FuncForPC(reflect.ValueOf(func() {}).Pointer()).Name())
}
```
outputs
```
main.main.func1
main.main.func2
main.myFunc
main.main.func3
```